utilities/src/main/java/io/syndesis/qe/resource/impl/SFTP.java

```diff
@@ -51,6 +51,7 @@ public void deploy() {
             .done();
         OpenShiftUtils.getInstance()
             .addRoleToUser("anyuid", "system:serviceaccount:" + TestConfiguration.openShiftNamespace() + ":" + serviceAccountName);
+        OpenShiftUtils.binary().execute("adm", "policy", "add-scc-to-group", "anyuid", "system:authenticated");
```

Comment:
this way you are changing the behavior for all users in the cluster while the sftp is running, so affecting other tests running in different namespaces.